js/jquery.flot.tooltip.source.js

```diff
@@ -286,10 +295,12 @@
             } else {
                 const existingTooltip = that.tooltips[0];
 
+                // If there's an existing sticky tooltip, we want to show the non-sticky one
                 if(that.tooltipOptions.stickyable && existingTooltip.isSticky) {
-                    $tooltipNode = existingTooltip.node;
+                    $tooltipNode = that.tooltips[1].node;
```

Comment:
I feel the opposite. I think we hover over a fixed tooltip then the hover one should not be shown. Having the hover one overlay the fixed tooltip looks very strange to me.